core/pom.xml

```diff
@@ -215,8 +215,8 @@
             <scope>test</scope>
         </dependency>
         <dependency>
-            <groupId>org.slf4j</groupId>
-            <artifactId>slf4j-reload4j</artifactId>
```

Comment:
is this already declared in top level pom of this project?

Reply:
I don't think this is declared in top level pom. Should I add it there? Maybe not all modules need this.